src/forms/TextInput.jsx

```diff
@@ -1,6 +1,7 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import cx from 'classnames';
+import { injectIntl, intlShape } from 'react-intl';
```

Comment:
Yea I ripped the IntlProvider our. I must have been advised to at some point but I'll change it.